exec/mailinglist.R

```diff
@@ -51,56 +47,62 @@ if(arguments[["tabulate"]] & arguments[["help"]]){
   conf <- yaml::read_yaml(conf_path)
 
   perceval_path <- path.expand(tool[["perceval"]])
-  mbox_path <- path.expand(conf[["mailing_list"]][["mbox"]])
-
-  project_mbox <- parse_mbox(perceval_path,mbox_path)
+  mbox_file_path <- path.expand(conf[["mailing_list"]][["mod_mbox"]][["project_key_1"]][["mbox_file_path"]])
 
-  cli_alert_success(paste0("Tabulated mailing list was saved at: ",save_path))
+  project_mbox <- parse_mbox(perceval_path, mbox_file_path)
 
-  data.table::fwrite(project_mbox,save_path)
-}else if(arguments[["download"]] & arguments[["modmbox"]] & arguments[["help"]]){
-  cli_alert_info("Saves a mailing list archive from mod_mbox as a .mbox file
-                 using download_mod_mbox().")
-}else if(arguments[["download"]] & arguments[["modmbox"]]){
+  data.table::fwrite(project_mbox, save_path)
+  cli::cli_alert_success(paste0("Tabulated mailing list was saved at: ", save_path))
 
+} else if (arguments[["download"]] & arguments[["modmbox"]] & arguments[["help"]]) {
+  cli::cli_alert_info("Downloads mailing list archives from mod_mbox using download_mod_mbox().")
+} else if (arguments[["download"]] & arguments[["modmbox"]]) {
 
   conf_path <- arguments[["<project_conf.yml>"]]
-  save_path <- arguments[["<save_file_name_path>"]]
-  conf <- yaml::read_yaml(conf_path)
+  start_year_month <- arguments[["<start_year_month>"]]
+  end_year_month <- arguments[["<end_year_month>"]]
+  save_folder_path <- arguments[["<save_folder_path>"]]
 
-  mod_mbox_url <- conf[["mailing_list"]][["domain"]]
-  mailing_list <- conf[["mailing_list"]][["list_key"]][1]
-
-  start_year <- arguments[["<start_year>"]]
-  end_year <- arguments[["<end_year>"]]
+  conf <- yaml::read_yaml(conf_path)
+  mailing_list <- conf[["mailing_list"]][["mod_mbox"]][["project_key_1"]][["mailing_list"]]
 
-  mbox <- download_mod_mbox(base_url = mod_mbox_url,
-                            mailing_list = mailing_list,
-                            from_year=start_year,
-                            to_year=end_year,
-                            save_file_path = save_path,
-                            verbose = TRUE)
+  download_mod_mbox(
```

Comment:
should use refresh. exec is for refresh only for downloaders that have them

Reply:
Changed to refresh